packages/core/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/core
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+export namespace Keys {
+  export class Context {
+    static readonly CONTROLLER_CLASS: string = 'controller.class';
+    static readonly CONTROLLER_METHOD_NAME: string = 'controller.method.name';
+    static readonly CONTROLLER_METHOD_META: string = 'controller.method.meta';
+  }
+
+  export class Authentication {
+    static readonly DECORATOR: string = 'authenticate';
```

Comment:
Maybe a better name is `AUTHENTICATION_META = 'authentication.meta'`. No `string` is needed as TS can infer it from the value.